deploy/examples/object.yaml

```
@@ -70,6 +70,13 @@ spec:
               # Use <topologyKey: failure-domain.beta.kubernetes.io/zone> in k8s cluster if your cluster is v1.16 or lower
               # Use <topologyKey: topology.kubernetes.io/zone>  in k8s cluster is v1.17 or upper
               topologyKey: kubernetes.io/hostname
+      # opsLogSidecar:
+      # resources:
+      #   limits:
+      #     memory: "60Mi"
+      #   requests:
+      #     cpu: "100m"
+      #     memory: "60Mi"
```

I think the indentation is off for the resources. If users use VScode to uncomment these, there will be an error applying the config I think

```suggestion
      # opsLogSidecar:
      #   resources:
      #     limits:
      #       memory: "60Mi"
      #     requests:
      #       cpu: "100m"
      #       memory: "60Mi"
```